<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditItemDialog</class>
 <widget class="QDialog" name="EditItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение\удаление</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Название</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Описание</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Дата и время</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
        margin: 0;
    background-color: rgb(33, 33, 33); /* Прозрачный фон */
    border: 2px solid rgb(33, 33, 33); /* Цвет рамки */
    color: rgb(239, 241, 247); /* Цвет текста */
    padding: 10px 20px; /* Отступы внутри кнопки */
    text-align: center; /* Выравнивание текста по центру */
    text-decoration: none; /* Убираем подчеркивание текста */
    font-size: 16px;
    border-radius: 0; /* Углы без закруглений */
}

QPushButton:hover {
    background-color: transparent; /* Новый цвет фона при наведении курсора */
    color: rgb(255, 255, 255); /* Изменение цвета текста при наведении */
    border-color: rgb(255, 255, 255); /* Изменение цвета рамки при наведении */
}
</string>
     </property>
     <property name="text">
      <string>Сохранить в файл</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="cancelBtn">
     <property name="styleSheet">
      <string notr="true">QPushButton {
        margin: 0;
    background-color: rgb(33, 33, 33); /* Прозрачный фон */
    border: 2px solid rgb(33, 33, 33); /* Цвет рамки */
    color: rgb(239, 241, 247); /* Цвет текста */
    padding: 10px 20px; /* Отступы внутри кнопки */
    text-align: center; /* Выравнивание текста по центру */
    text-decoration: none; /* Убираем подчеркивание текста */
    font-size: 16px;
    border-radius: 0; /* Углы без закруглений */
}

QPushButton:hover {
    background-color: transparent; /* Новый цвет фона при наведении курсора */
    color: rgb(255, 255, 255); /* Изменение цвета текста при наведении */
    border-color: rgb(255, 255, 255); /* Изменение цвета рамки при наведении */
}</string>
     </property>
     <property name="text">
      <string>Отменить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="editBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton {
        margin: 0;
    background-color: rgb(33, 33, 33); /* Прозрачный фон */
    border: 2px solid rgb(33, 33, 33); /* Цвет рамки */
    color: rgb(239, 241, 247); /* Цвет текста */
    padding: 10px 20px; /* Отступы внутри кнопки */
    text-align: center; /* Выравнивание текста по центру */
    text-decoration: none; /* Убираем подчеркивание текста */
    font-size: 16px;
    border-radius: 0; /* Углы без закруглений */
}

QPushButton:hover {
    background-color: transparent; /* Новый цвет фона при наведении курсора */
    color: rgb(34, 255, 39); /* Изменение цвета текста при наведении */
    border-color: rgb(34, 255, 39); /* Изменение цвета рамки при наведении */
}</string>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton {
        margin: 0;
    background-color: rgb(33, 33, 33); /* Прозрачный фон */
    border: 2px solid rgb(33, 33, 33); /* Цвет рамки */
    color: rgb(239, 241, 247); /* Цвет текста */
    padding: 10px 20px; /* Отступы внутри кнопки */
    text-align: center; /* Выравнивание текста по центру */
    text-decoration: none; /* Убираем подчеркивание текста */
    font-size: 16px;
    border-radius: 0; /* Углы без закруглений */
}

QPushButton:hover {
    background-color: transparent; /* Новый цвет фона при наведении курсора */
    color: rgb(255, 15, 21); /* Изменение цвета текста при наведении */
    border-color: rgb(255, 15, 21); /* Изменение цвета рамки при наведении */
}</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>EditItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
